app/serializers/error_serializer.rb

```diff
@@ -0,0 +1,26 @@
+class ErrorSerializer
+  def initialize(error_object)
+    @error_object = error_object
+  end
+
+  def serialize_json
+    {
+      errors: [
+        {
+          status: @error_object.status_code.to_s,
+          title: @error_object.message
+        }
+      ]
+    }
+  end
+
+  def serializer_validation
+    {
+      errors: [
+          {
+            detail: "Validation failed: #{@error_object.hash_message_to_s.first}"
+          }
```

Comment:
I did some research and I think we can create another rescue using `ActiveRecord::RecordInvalid` I will try this in my branch and see if I can generate the message from there